packages/voting-module-adapter/adapters/cw20-staked-balance-voting/components/Membership/index.tsx

```diff
@@ -83,6 +86,18 @@ const InnerMembership: FC<Omit<MembershipProps, 'primaryText'>> = ({
   const [showStakingMode, setShowStakingMode] = useState<StakingMode>()
   const stakingLoading = useRecoilValue(stakingLoadingAtom)
 
+  // Get max proposal deposit that needs the governance token.
```

Comment:
Let's make this comment more specific by telling the what and why of what we're doing. Something like:

```
// Of all the proposal modules this DAO uses, find the highest proposal deposit.
// We'll suggest that the user reserve that amount for proposals in our staking modal.
```